pkgs/by-name/cl/clatd/package.nix

```diff
@@ -0,0 +1,58 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, makeWrapper
+, perl
+, perlPackages
+, tayga
+, iproute2
+, iptables
+}:
+
+stdenv.mkDerivation rec {
+  pname = "clatd";
+  version = "1.6";
+
+  src = fetchFromGitHub {
+    owner = "toreanderson";
+    repo = "clatd";
+    rev = "v${version}";
+    hash = "sha256-ZUGWQTXXgATy539NQxkZSvQA7HIWkIPsw1NJrz0xKEg=";
+  };
+
+  nativeBuildInputs = [ makeWrapper ];
+  buildInputs = with perlPackages; [
+    perl
+    NetIP
+    NetDNS
+  ];
+
+  dontBuild = true;
+
+  installPhase = ''
+    runHook preInstall
+
+    install -Dm0755 --target-directory=$out/bin clatd
+    patchShebangs $out/bin/clatd
+    wrapProgram $out/bin/clatd \
```

Comment:
This should be done in `postFixup`. Also why not using upstream Makefile here?

Reply:
Changed to use upstream Makeflie and move our logic to postFixup, thanks!